### DISCUSSION TITLE
Layouts RFC

### DISCUSSION THREAD

**Discussion by Bowriverstudio** [wpengine/faustjs#1101] [Ideas]
Hello

When I look at the "Apollo Faust Starter" it is not using `Layouts RFC`  as described in https://nextjs.org/blog/layouts-rfc or in this conversation.

[Discord convertation ](https://discord.com/channels/836253505944813629/984134897381822485/1029373055790759936)

Ideally, I'd like to see a sample of this as most of my WordPress sites use the same general Layout.

Thank you for your consideration.

**Comment by theodesp** [comment]
Hello @Bowriverstudio. Thank you for the suggestion. We are aware of this Layouts RFC and we anticipate no issues with it and Faust.

However we cannot provide an example yet as the status of the RFC is, well, unknown and there is no official guide for testing it out.

Hopefully we will get more updates next week in Next.js Conf.

https://github.com/vercel/next.js/discussions/37136#discussioncomment-3907990
